### PR TITLE
Button style changes:

### PR DESCRIPTION
**Pull request by wittejm**
* Expanded pill padding.
 * Shadow was cut off by the container boundary because the button width is 100%. Added padding to make room for the shadow.
 * Removed weird white trim line caused by borderwidth (which was the same color as background).
 * Removed shadow from disabled buttons.
 * Color for disabled button already looks correct.
 * Sticky button: I don’t see how to support this. Suggestions I see online are just to put the “sticky” component below a ScrollView. Added the example to the Button Storybook.

### CODE REVIEW

**Comment by wittejm** [comment]
[sc-825]

File: src/UserInput/Button.react.tsx [hunk 1]
```diff
@@ -106,14 +114,13 @@ const Button: React.FC<ButtonProps> = (props: ButtonProps) => {
...
       getBackgroundStyle(),
-      props.noGrow ? { width: undefined } : {},
+      props.noGrow ? { ...Spacing.paddingVertical, ...Spacing.largePaddingHorizontal, width: undefined } : {},
       props.nav ? { borderRadius: 19, height: 40 } : {},
       props.buttonStyle,
     ],
-    disabledStyle: props.secondary ? Styles.secondaryDisabledBackground : Styles.primaryDisabledBackground,
-    disabledTitleStyle: props.secondary ? {color: Colors.RED_200} : { color: Colors.WHITE },
+    disabledStyle: props.rank && props.rank in ["secondary", "tertiary"] ? Styles.secondaryDisabledBackground : Styles.primaryDisabledBackground,
+    disabledTitleStyle: props.rank && props.rank in ["secondary", "tertiary"] ? { color: Colors.RED_200 } : { color: Colors.WHITE },
```
Comment (cainlevy) [inline src/UserInput/Button.react.tsx]: would checking for a primary rank be simpler than `props.rank in ["secondary", "tertiary"]?

it strikes me that primary is the default rank both here and in `getBackgroundStyle`. we might benefit from resolving that default early and eliminating the null case forever after.

**Review by cainlevy — APPROVED** [review]

File: src/UserInput/Button.react.tsx (hunk 1 above)
Comment (wittejm) [reply src/UserInput/Button.react.tsx]: Good point. Change applied.

**Comment by wittejm** [comment]
[sc-792]

**Pull request merged**